elements/pfe-tabs/src/pfe-tab-panel.scss

```diff
@@ -92,3 +113,34 @@ $LOCAL-VARIABLES: (
 :host([hidden]) {
   display: none;
 }
+
+/// ===========================================================================
+/// CONTEXTS: DARK & SATURATED
+/// ===========================================================================
+/// In dark & saturated contexts, we override the local color vars
+
+
+:host {
+  @at-root #{&}([pfe-variant="earth"][on="dark"]) {
+    --pfe-tabs__panel--BackgroundColor: #{pfe-var(surface--darkest)};
+  }
+}
+
+:host {
+  @at-root #{&}([pfe-variant="earth"][on="saturated"]) {
+    --pfe-tabs__panel--BackgroundColor: #{pfe-var(surface--lightest)};
+
+      ::slotted(*) {
+        color: pfe-var(text);
+      }
+  }
+ 
+}
+
+//VERTICAL
```

Comment:
Vertical styles are listed above on line 84.  Don't need the at-root syntax here either.

Reply:
updated